appveyor.yml

```diff
@@ -6,14 +6,13 @@ environment:
   - TOXENV: "py34-xdist"
   - TOXENV: "py35-xdist"
   - TOXENV: "py36-xdist"
+  # NOTE: pypy-xdist is buggy currently (https://github.com/pytest-dev/pytest-xdist/issues/142).
   - TOXENV: "pypy"
     PYTEST_NO_COVERAGE: "1"
   # Specialized factors for py27.
   - TOXENV: "py27-trial,py27-numpy,py27-nobyte"
-  - TOXENV: "py27-pluggymaster"
```

Comment:
I would prefer not to touch this; when we adopt Azure for good, we will disable AppVeyor entirely, so it doesn't make much sense to change things mid way now.

Reply:
This is in features already: https://github.com/pytest-dev/pytest/commit/dc8c27037a7e6e6cd24dd5e3fd2ee6bc56db3594

Reply:
Oh nevermind then! 😋 

Besides https://github.com/pytest-dev/pytest/issues/4807, we could adopt Azure as our official Windows build right away. I will try to get to it later.